custom_parameters.properties

```diff
@@ -0,0 +1,2 @@
+index_ancestors = true
+index_compound_sequence = false
```

Comment:
I realize this is probably desired in _this_ environment... really, _most_ environments, but... when pulled out for general use, should we make it be `true`, to maintain legacy behaviours? Maybe? Dunno...

Reply:
Afraid so.

Reply:
Being done across:
* https://github.com/discoverygarden/basic-solr-config/pull/196/commits/059cdf5c43793dfc02d69feaed990c6dfe3db15a
* https://github.com/discoverygarden/basic-solr-config/pull/197/commits/90a6f9cbcd8a253eed97c600d668129f7b7970ed and
* https://github.com/discoverygarden/basic-solr-config/pull/198/commits/0c706d8f0ba67f5c988c82067b0ccdcf916ab4f6